src/curated-corpus/components/MiniScheduleCard/MiniScheduleCard.tsx

```diff
@@ -59,30 +60,23 @@ export const MiniScheduleCard: React.FC<MiniScheduleCardProps> = (
               <LabelOutlinedIcon fontSize="small" />
             </ListItemIcon>
             <ListItemText
-              className={classes.listItemText}
-              secondary={item.approvedItem.topic.toLowerCase()}
+              secondary={getDisplayTopic(item.approvedItem.topic)}
             />
           </ListItem>
           {item.approvedItem.isSyndicated && (
             <ListItem disableGutters>
               <ListItemIcon className={classes.listItemIcon}>
                 <CheckCircleOutlineIcon fontSize="small" />
               </ListItemIcon>
-              <ListItemText
-                className={classes.listItemText}
-                secondary={'Syndicated'}
-              />
+              <ListItemText secondary={'Syndicated'} />
```

Comment:
Also removed a CSS style that was capitalising the first letter of the topic - this is no longer needed. Before this change, the component would take the topic code - all uppercase - for example, `TECHNOLOGY`, transform it into lowercase with JS and capitalise the first letter with CSS 🤦. It worked when topic codes in the shared data had symbols and spaces, e.g. `HEALTH & FITNESS`, but definitely didn't when the shared data was updated to use underscores instead, e.g. `HEALTH_FITNESS`.